src/main/client/src/components/@pages/AchievementPage.tsx

```diff
@@ -144,8 +155,23 @@ export const AchievementPage: React.FC = (props) => {
     </Box>
   )
 
+  const breadcrumbItems = [
+    {
+      title: 'Bucketlist',
+      to: '/bucketlist'
+    },
+    {
+      title: categoryName,
+      to: `/bucketlist/kategoria/${achDetails.achievement?.categoryId}`
+    },
+    {
+      title: achDetails.achievement?.title
+    }
+  ]
+
   return (
     <Page {...props} loginRequired>
+      <CustomBreadcrumb items={breadcrumbItems} />
       <Heading mb={0}>{achDetails.achievement?.title}</Heading>
       <AchievementStatusBadge status={achDetails.status} fontSize="lg" />
```

Comment:
Top margin of 5

Reply:
I forgot about the margins this time and I also forgot to check the uploaded file size before submission, so I'll come back to this tomorrow.